Essentials/src/main/resources/messages.properties

```diff
@@ -675,21 +675,27 @@ loomCommandDescription=Opens up a loom.
 loomCommandUsage=/<command>
 mailClear=\u00a76To clear your mail, type\u00a7c /mail clear\u00a76.
 mailCleared=\u00a76Mail cleared\!
+mailClearedPlayer=\u00a76Mail of {0} cleared!\
+mailClearedAll=\u00a76All players' mail cleared!\
 mailClearIndex=\u00a74You must specify a number between 1-{0}.
 mailCommandDescription=Manages inter-player, intra-server mail.
-mailCommandUsage=/<command> [read|clear|clear [number]|send [to] [message]|sendtemp [to] [expire time] [message]|sendall [message]]
+mailCommandUsage=/<command> [read|clear|clear [number]|clear [player]|send [to] [message]|sendtemp [to] [expire time] [message]|sendall [message]|clearall]
 mailCommandUsage1=/<command> read [page]
 mailCommandUsage1Description=Reads the first (or specified) page of your mail
 mailCommandUsage2=/<command> clear [number]
 mailCommandUsage2Description=Clears either all or the specified mail(s)
-mailCommandUsage3=/<command> send <player> <message>
-mailCommandUsage3Description=Sends the specified player the given message
-mailCommandUsage4=/<command> sendall <message>
-mailCommandUsage4Description=Sends all players the given message
-mailCommandUsage5=/<command> sendtemp <player> <expire time> <message>
-mailCommandUsage5Description=Sends the specified player the given message which will expire in the specified time
-mailCommandUsage6=/<command> sendtempall <expire time> <message>
-mailCommandUsage6Description=Sends all players the given message which will expire in the specified time
+mailCommandUsage3=/<command> clear [player]
+mailCommandUsage3Description=Clears the mail of the specified player.
+mailCommandUsage4=/<command> send <player> <message>
+mailCommandUsage4Description=Sends the specified player the given message
+mailCommandUsage5=/<command> sendall <message>
+mailCommandUsage5Description=Sends all players the given message
+mailCommandUsage6=/<command> sendtemp <player> <expire time> <message>
+mailCommandUsage6Description=Sends the specified player the given message which will expire in the specified time
+mailCommandUsage7=/<command> sendtempall <expire time> <message>
+mailCommandUsage7Description=Sends all players the given message which will expire in the specified time
+mailCommandUsage8=/<command> clearall
+mailCommandUsage8Description=Clears the mail of all players
```

Comment:
Right now this will delete existing translations in other languages. `clear [player]` should be added at the bottom, between `sendtempall` and `clearall`.